### PR TITLE
Clean up branch

### PR DESCRIPTION
**Pull request by eskehaack**

### CODE REVIEW

**Review by Mikkelthe — COMMENTED** [review]
Word_eval delen:
README empty - kan jeg godt fikse efter
Er vi helt 100 på det vi har gjort hedder Zero-shot, fordi jeg er ikke så sikker på det når jeg har læst om zero-shot

**Comment by Magdk01** [comment]
> 

Jeg tror også vi skal bruge et andet navn

File: README.md
Comment (Magdk01) [inline README.md:1]: Should we not have a item list for the steps like: 
1. Do preprocess
2. train
3. ...

**Review by Magdk01 — COMMENTED** [review]
I think most looks fine, but would love to have a step by step added to the main readme.

If nothing else, i will add this later myself if you agree.

File: README.md
Comment (Mikkelthe) [reply README.md:1]: Except for the preprocessing of data every folder except for word eval can be ran in any order no?

**Comment by Magdk01** [comment]
A bit more cleanup. We now need to rewrite the README in sentiment_evaluation, before closing.

**Comment by Magdk01** [comment]
As this is currently the upstream branch i took the liberty to merge my feature branch here.

File: transfer_learning/Quarterly_trainer.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def group_texts(examples):
     )
 
     
-    if conti_nue == "continue":
+    if conti_nue ==  :
```
Comment (Magdk01) [inline transfer_learning/Quarterly_trainer.py]: Search and replace, fucked this. I will fix it.

**Review by Magdk01 — APPROVED** [review]

**Pull request merged**